tunelinux/iscsi.go

```diff
@@ -417,3 +422,41 @@ func ConfigureIscsi() (err error) {
 
 	return nil
 }
+
+// Update replacement timeout for logged-in sessions
+func UpdateIscsiSessionReplacementTimeout() (err error) {
```

Comment:
I think we can use generic function to update this like any other parameter. We don't need separate one for each parameter to apply.